src/redux/reducers/profileReducer.js

```diff
@@ -34,6 +34,13 @@ const profileReducer = (state = initialState, action) => {
       };
     }
 
+    case ACTION_TYPES.SAVE_PHOTO_SUC: {
+      return {
+        ...state,
+        profile: {...state.profile, photos: action.photos },
```

Comment:
Are you sure about "action.photos"?
It seems you should get photos from "payload"

Reply:
sure